plugins/thunderbolt/README.md

```diff
@@ -57,6 +57,33 @@ If a user sets `DelayedActivation` configuration option then the update will
 be staged but not completed until `activate` is separately called such as at
 logout or shutdown.
 
+## USB4 Retimer Fun
```

Comment:
Maybe a different word than `Fun` in the readme.

Reply:
Indeed, fixed.